pyproject.toml

```diff
@@ -11,7 +11,7 @@ readme = "DESCRIPTION.md"
 python = "^3.8"
 textstat = "^0.7.3"
 pandas = "*"
-whylogs = "^1.2.4"
+whylogs = "^1.2.5"
```

Comment:
I don't think we can unyank 1.2.5, but I think it will still use it until 1.2.6 is released.

Reply:
whylogs 1.2.6 is live: https://pypi.org/project/whylogs/1.2.6/